docs/src/main/asciidoc/_templates/doc-concept.adoc

```diff
@@ -0,0 +1,80 @@
+////
```

Comment:
This file should be removed. The _templates folder should only contain templates to be copied.

Reply:
@ebullient Thanks! Agreed! ...and removed.

I also squashed the "fix-up" commits into the first one.